config/environments/development.rb

```diff
@@ -66,4 +66,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # Docker/Vagrant debugging
+  config.web_console.whitelisted_ips = ['172.16.0.0/12', '192.168.0.0/16', '10.0.0.0/8']
```

Comment:
Also here. Same question.

Reply:
@bkeese I lean toward saying this is universally a good change.  I basically means that Docker and Vagrant debugging works out to the box.  Having said that it probably isn't directly related to the scope of this PR, so I've taken it out of here. If y'all want it in separately, I do recommend it though.

Reply:
@phuongdh Want to weigh in on this one? I'm in favor of keeping it now that I know what it's about.

Reply:
@bkeese if @phuongdh wants it in I can make it part of the second PR

Reply:
Looks good. :+1: to keeping it through 2nd PR.